src/colors.ts

```diff
@@ -0,0 +1,5 @@
+import dark from './dark'
```

Comment:
Maybe better if you want to support treeshaking in the future would be
```js
export {default as dark} from './dark'
export {default as light} from './light'
export {default as mirage} from './mirage'
```
But it's nothing important.

Reply:
This syntax isn't supported by TypeScript since this syntax is still on stage 1.